### DISCUSSION TITLE
afethukuk.com - Afetzedelerin adalete erişimini kolaylaştırmayı amaçlayan platform.

### DISCUSSION THREAD

**Discussion by AlperenMecit** [acikkaynak/deprem-yardim-projesi#50] [Ideas]
Merhabalar,

Ben Alperen Mecit, eski bir avukatım şu anda bir startupta ekosistem müdürü olarak çalışıyorum.

afethukuk.com fikrini bugün değerli arkadaşım Elif Sena Yaman bana iletti ve üstüne hızlıca çalışmaya başladık. Ankara Barosu Başkanı ile görüşmeyi gerçekleştirdik ve görev paylaşımımızı yaptık. Şu anda saat 01.27, Ebru Metin, Elif Sena Yaman ve ben toplantıdayız ve size bu fikir önerisini yazıyorum.

Neredeyse her yıl ayrı bir afet yaşanan ülkemizde, kişilerin afet kaynaklı hukuki hak kayıplarını engellemeye çalışan afethukuk.com'da:

- Afet döneminde dezavantajlı kesimlerin adalete erişiminde rehberlik sağlanması
- Afetzedelerin hukuki işlemlerinde destek alabilmeleri adına pro-bono ağının kurulması
- Afetzede hukukçulara istihdam yaratılması
- Afetzede hukuk öğrencilerinin kariyer gelişimlerinin desteklenmesi
- Afet konusunda çalışmayı hedefleyen hukukçuları bir araya getirilmesi üzerine çalışacağız.

Kişilerin afethukuk.com'a ulaşabilmesi için teknik desteğe ihtiyacımız var. Bu konuda beraber çalışmak isteyenlere açık çağrımızdır.

alperenceyhunmecit@hotmail.com

**Comment by AlperenMecit** [comment]
Ankara Barosu ile:
-Afet sonrası acilen yapılması gereken hukuki işlemlere dair bir kılavuz ve
-Afet alanına yardım için gidecek ekiplere verilecek eğitim kitapçıklarının tasarlanması işlerine başladık.

Çocukları kaybolan kişilerin nereye, nasıl başvuracağı, sigorta başvurularının nasıl ve ne zaman yapılacağı gibi konularda da avukat ve akademisyenlerle çalışarak içerikler üreteceğiz.

**Reply by dengizik** [reply]
Merhaba, 

bir de enkazlardan örnek alma durumu var. Bununla ilgili bir çalışma başlatıldı mı, biliyor musunuz? Ben şöyle bir özellik önerdim afetharita.com'a: [https://github.com/acikkaynak/deprem-yardim-projesi/discussions/84 ]

**Reply by AlperenMecit** [reply]
Merhaba,

TBB bünyesinde bir çalışma var. Delil tespiti amacıyla örnek alma durumu usulüne uygun yapılmadığı sürece mahkeme karşısında delil niteliği taşımıyor. Bu nedenle Adalet Bakanlığı ve Barolar Birliğinin aldığı teknolojik adımlar var diye biliyorum. Şu anda afethukuk.com'da afetzedeler acil ihtiyaçlarına karşılık vermeye çalışıyoruz. Her ne kadar örnek alma durumu önemli olsa da, doğrudan fayda sağlayamadığımız için başlamadık. Bu alanda çalışan herkese destek vermeye açığız.

**Reply by dengizik** [reply]
anladım, teşekkür ederim.

**Comment by ramazansancar** [comment]
Merhaba arkadaşlar,
Öncelikle fikriniz için teşekkürler. Bu talebiniz Legal ekibin kontrolünden geçti.
PM ekibine fikir iletildi.
